source/turtles.rst

```diff
@@ -157,7 +165,7 @@ Code in files
 
 Most code is written and executed from a file. 
 
-Using SublimeText create a file named `my_turtle_file.py` with this code:: 
+Using SublimeText to create a file named `my_turtle_file.py` with this code:: 
```

Comment:
Doesn't state whether sublime text is on there, I am going to merge this though and open an issue to test.